### PR TITLE
Fix root7 pretty function

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

File: core/base/v7/inc/ROOT/TLogger.hxx [hunk 1]
```diff
@@ -113,8 +113,16 @@ public:
 } // namespace Experimental
 } // namespace ROOT
 
+#if defined(_MSC_VER)
+#define LOG_PRETTY_FUNCTION __FUNCSIG__
```
Comment (pcanal) [inline core/base/v7/inc/ROOT/TLogger.hxx]: Can you prefix the macro with R__ do reduce the risk of clash with user macros?

Thanks,
Philippe.

File: core/base/v7/inc/ROOT/TLogger.hxx [hunk 2]
```diff
@@ -113,8 +113,16 @@ public:
...
+#elif defined(__GNUC__) || defined(__clang__)
```
Comment (Axel-Naumann) [inline core/base/v7/inc/ROOT/TLogger.hxx]: I'd rather leave this the default: it's more likely that we miss a compiler here (eg ICC) than that compiler not supporting `__PRETTY_FUNCTION__`... Ie can you remove the if here and the func part below? We will immediately see if a compiler does not support this. Thanks!

Comment (bellenot) [reply core/base/v7/inc/ROOT/TLogger.hxx]: Done

File: core/base/v7/inc/ROOT/TLogger.hxx (hunk 1 above)
Comment (bellenot) [reply core/base/v7/inc/ROOT/TLogger.hxx]: Done

**Comment by bellenot** [comment]
I suppose I can merge it now. If not, I'll make required changes after the merge

**Pull request merged**